Gemfile.lock

```diff
@@ -0,0 +1,166 @@
+GIT
```

Comment:
i was just building against master, should i use rubygem release instead? I'll nuke the lock file once the review is done, i'll do one more round of testing against the rubygem version then,